src/test/resources/test-schema.json

```diff
@@ -30,6 +30,10 @@
             }
         }
     },
+    "propertyTransform": {
+        "/properties/propertyA": "$join([$string(test), propertyA])",
+        "/properties/propertyB": "$count(propertyB) = 0 ? null : propertyB"
```

Comment:
Out of curiosity:
- which language is this?
- Is that information documented somewhere?

Reply:
Agree with this let's try to document this a bit better in what it actually means. All other schema properties have additional documentation about how exactly they function.

Reply:
Out of curiosity:

which language is this?
> This is jsonta: https://jsonata.org/
Is that information documented somewhere?
> I am working on adding more documentation for this soon. Should be able to update this package's README and the CFN public documentation for the same soon.